api/src/Entity/User.php

```diff
@@ -126,13 +138,37 @@ class User extends BaseEntity implements UserInterface, PasswordAuthenticatedUse
     #[Groups(['read', 'write'])]
     public ?string $language = null;
 
+    /**
+     * The state of this user.
+     *
+     * @ORM\Column(type="string", length=16, nullable=false)
+     */
+    #[ApiProperty(readable: false, writable: false)]
+    public string $state = self::STATE_NONREGISTERED;
```

Comment:
Müssten wir diesen neuen State des Users in den security rules und in den entity filtern miteinbeziehen? Kann jemand der noch nicht aktiviert ist oder jemand der gelöscht ist Camps, Activities etc. lesen? Schreiben? Welche Auswirkung hat die Mail-Validierung und dieser Status überhaupt?